manifests/zone.pp

```diff
@@ -42,10 +45,11 @@
   validate_re($ensure, ['present', 'absent'],
               "\$ensure must be either 'present' or 'absent', got '${ensure}'")
 
-  validate_bool($is_dynamic)
```

Comment:
this one should still be present I guess — we should remove it only once we remove the "is_slave" variable.